http_check/datadog_checks/http_check/http_check.py

```diff
@@ -100,6 +100,7 @@ def send_status_down(loginfo, down_msg):
 
         # Store tags in a temporary list so that we don't modify the global tags data structure
         tags_list = list(tags)
+        tags_list.append('status_code:{}'.format(http_response_status_code))
```

Comment:
`http_response_status_code` values is https://github.com/DataDog/integrations-core/blob/3aabbae5beeec60e8f99c9e7c0f87b44bab02cbe/http_check/datadog_checks/http_check/data/conf.yaml.example#L194-L198